<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComPortPropertiesWindow</class>
 <widget class="QDialog" name="ComPortPropertiesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblPortName">
       <property name="text">
        <string>Port Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxPortName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblBaud">
       <property name="text">
        <string>Baud Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxBaud"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblDataBits">
       <property name="text">
        <string>Data Bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxDataBits"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblParity">
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxParity"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lblStopBits">
       <property name="text">
        <string>Stop Bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxStopBits"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pbDisconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="pbRelativeMove">
       <property name="text">
        <string>Relative Move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetAsOrigin">
       <property name="text">
        <string>Set as Origin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAbsoluteMove">
       <property name="text">
        <string>Absolute Move</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="pbUnits_mm">
       <property name="text">
        <string>Units mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbUnits_in">
       <property name="text">
        <string>Units in</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
